solutions/bronze/cf-1808B.mdx

```diff
@@ -0,0 +1,61 @@
+---
+id: cf-1808B
+source: CF 1808B
+title: Playing in a Casino
+author: Amit Singh Bisht
+---
+
+## Explanation
+
+Essentially, the solution is to calculate the difference between the sum of the larger elements and the sum of the smaller
+elements in each column, and weighting this difference by the number of times that the smaller element appears in that column.
+This is based on the observation that if we have n numbers, and we want to minimize the sum of absolute differences between
+each number and some target value, then the target value should be the median of those n numbers. This solution applies this
+observation to each column of the matrix, separately.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2 M)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios_base::sync_with_stdio(false);
+	cin.tie(NULL);
+	int t;
+	cin >> t;
+	for (int i = 0; i < t; i++) {
+		int n, m;
+		cin >> n >> m;
+		vector<vector<int>> a(n, vector<int>(m));
+		for (int i = 0; i < n; i++) {
+			for (int j = 0; j < m; j++) { cin >> a[i][j]; }
+		}
+
+		int ans = 0;
+		for (int i = 0; i < m; i++) {
+			vector<int> temp;
+			int sum = 0.0;
```

Comment:
why 0.0

Reply:
that makes a lot of sense. I used your code forces solution in my commit bc it's too late at night rn for me to think if that's fine.

Reply:
>the problem lies in converting everything to ints. convert only the variables that need to be.

that makes a lot of sense. I used your codeforces solution in my newest commit because its too late at night for me to think rn if that's fine.